packages/components/dashboard/dashboard.scss

```diff
@@ -18,6 +18,10 @@
   display: flex;
 }
 
+.co-dashboard-body .pf-c-card {
```

Comment:
I am not aware of this format (2 class names with a space in between), is it a form of descendant combinator ??